elements/pfe-spinner/demo/pfe-spinner.html

```diff
@@ -0,0 +1,36 @@
+<link rel="stylesheet" href="/core/pfe-styles/pfe.min.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-base.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-context.css">
+<link rel="stylesheet" href="/core/pfe-styles/pfe-layouts.css">
+<link rel="stylesheet" href="./demo.css">
+<script type="module" src="./pfe-spinner.js"></script>
+
+<pfe-band color="lightest">
```

Comment:
better not to use pfe-band

it'll be replaced shortly, and anyways we can just use div or section here (colour context is an rhds feature)

Reply:
I think I'm going to leave pfe-band for now until we have something we can apply consistently to all the demo pages.